design-toolkit/components/src/components/drawer/index.tsx

```diff
@@ -117,38 +119,60 @@ function DrawerLayout({
 DrawerLayout.displayName = 'Drawer.Layout';
 DrawerLayout.Main = DrawerLayoutMain;
 
+const tooltipPlacementMap = {
+  left: 'right',
+  right: 'left',
+  top: 'bottom',
+  bottom: 'top',
+} as const;
+
 function DrawerMenuItem({
   for: id,
   children,
   className,
   toggle,
   views,
+  textValue,
   ...rest
 }: DrawerMenuItemProps) {
   const { parent, stack } = useContext(ViewStackContext);
+  const { placement } = useContext(DrawerContext);
   const view = stack.at(-1);
   const action = toggle ? 'toggle' : 'open';
+  const tooltipRef = useRef(null);
 
   if (!parent) {
     return null;
   }
 
   return (
-    <DrawerTrigger for={`${action}:${id}`}>
-      <ToggleButton
-        {...rest}
-        className={composeRenderProps(className, (className) =>
-          item({ className }),
-        )}
-        role='tab'
-        variant='icon'
-        isSelected={id === view || !!views?.some((view) => id === view)}
+    <Tooltip isDisabled={!textValue}>
```

Comment:
Considering we're intending the menu items to be icons by design, I think we should require the `textValue`